backend/src/middlewares/upload-file/upload-file.middleware.ts

```diff
@@ -0,0 +1,11 @@
+import multer from 'multer';
+
+const storage = multer.memoryStorage();
+
+const upload = multer({
+  storage,
+});
+
+const uploadFile = upload.single('file');
```

Comment:
```suggestion
const UPLOAD_FILE_KEY = 'file';

const uploadFile = upload.single(UPLOAD_FILE_KEY);
```
we'll use this in both apps

move const to the `shared/src/common/constants/file`